docs/install.md

```diff
@@ -88,7 +88,7 @@ It is possible to install official containerlab releases via public APT/YUM repo
 /// tab | APT
 
 ```bash
-echo "deb [trusted=yes] https://apt.fury.io/netdevops/ /" | \
+echo "deb [trusted=yes] https://netdevops.fury.site/apt/ /" | \
 sudo tee -a /etc/apt/sources.list.d/netdevops.list
 
 sudo apt update && sudo apt install containerlab
```

Comment:
lets also change the yum URL, as they suffer from the same issue 

the new url: `https://netdevops.fury.site/yum/`